packages/geo/src/lib/layer/shared/layers/layer.interface.ts

```diff
@@ -63,6 +63,7 @@ export interface GeoWorkspaceQueryOptions {
 
 export interface LayersLink {
   linkId: string;
+  showInMiniBaseMap?: boolean; // default value is true
```

Comment:
p-etre faire un réel commentaire
/* 
  * Default value to true
  */

dans les IDE, on pourrait le voir directement!